### PR TITLE
Revert "Merge pull request #1171 from alphagov/global-banner"

### PR DESCRIPTION
**Pull request by boffbowsh**
This reverts commit bf74dd4a6a893cefabb72e4987097e463a179530, reversing
changes made to 41e4be34ff554edc8ec1dd9b91c235890fe1b13a.

This reverts commit bf74dd4a6a893cefabb72e4987097e463a179530, reversing
changes made to 41e4be34ff554edc8ec1dd9b91c235890fe1b13a.

Reverts #1171 and #1208

### CODE REVIEW

File: app/assets/stylesheets/views/_homepage.scss [hunk 1]
```diff
@@ -11,31 +11,60 @@ body.homepage {
   }
 }
 
-#homepage-promo-banner {
```
Comment (fofr) [inline app/assets/stylesheets/views/_homepage.scss]: I'm not sure why this diff isn't just a red block removing everything within `#homepage-promo-banner`.

Comment (boffbowsh) [inline app/assets/stylesheets/views/_homepage.scss]: I agree. Diffing is hard. The result seems correct though to my non CSS brain

Comment (boffbowsh) [inline app/assets/stylesheets/views/_homepage.scss]: Oh wait, no, you're right. This is probably reintroducing recently removed stuff. I'll take another look.

**Comment by leenagupte** [comment]
@boffbowsh If all of the promo banner stuff is going, I'd remove the Emergency Banner related stuff too. All it does is determine whether the promo banner can be displayed. It's not being used anywhere else: 
https://github.com/alphagov/frontend/search?utf8=%E2%9C%93&q=Emergency&type=

File: app/assets/stylesheets/views/_homepage.scss (hunk 1 above)
Comment (boffbowsh) [inline app/assets/stylesheets/views/_homepage.scss]: Fixed.

File: app/assets/stylesheets/views/_homepage.scss [hunk 2]
```diff
@@ -11,31 +11,60 @@ body.homepage {
...
+  p,
+  a {
+    color: $foreground;
+  }
+}
 
-    p {
-      margin: 0;
-      padding: $gutter-two-thirds 0;
+#campaign {
```
Comment (leenagupte) [inline app/assets/stylesheets/views/_homepage.scss]: I think the campaign styles should have been removed in the 2-week blitz. Oops.

**Comment by boffbowsh** [comment]
Reverted #1208 too @leenagupte

**Pull request merged**